qiskit/optimization/problems/linear_constraint.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019, 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Linear Constraint."""
+
+from typing import Union, List, Dict
+
+from numpy import ndarray
+from scipy.sparse import spmatrix
+
+from qiskit.optimization.problems.constraint import Constraint
+from qiskit.optimization.problems.linear_expression import LinearExpression
+
+
+class LinearConstraint(Constraint):
+    """ Representation of a linear constraint."""
+
+    def __init__(self,
+                 quadratic_program: "QuadraticProgram", name: str,
+                 linear: Union[ndarray, spmatrix, List[float], Dict[Union[str, int], float]],
+                 sense: Constraint.Sense,
+                 rhs: float
+                 ) -> None:
+        """Constructs a linear constraint.
```

Comment:
Again these constructs X when its for the init of a class X are not saying anything the code does not already so if we can sort these out in this pkg too.